.travis.yml

```diff
@@ -4,18 +4,26 @@ language: cpp
 services:
   - docker
 
-before_install:
-  - docker pull boxdot/flatdata-dev:15
-
 env:
-  - C_COMPILER=clang CXX_COMPILER=clang++
-  - C_COMPILER=gcc CXX_COMPILER=g++
+  global:
+    - DOCKER_IMAGE=boxdot/flatdata-dev:15
+  matrix:
+    - C_COMPILER=clang CXX_COMPILER=clang++
+    - C_COMPILER=gcc CXX_COMPILER=g++
+
+before_install:
+  - docker pull $DOCKER_IMAGE
 
 script:
   - |
     docker run -t \
       -e C_COMPILER=${C_COMPILER} \
       -e CXX_COMPILER=${CXX_COMPILER} \
       -v $(pwd):/src \
+      -v $(pwd)/build:/build \
       -w /build \
-      boxdot/flatdata-dev /src/ci/build-and-test.sh
+      $DOCKER_IMAGE /src/ci/build-and-test.sh
+  - sudo chown -R travis:travis $(pwd)/build
```

Comment:
Are you sure this will work? You are not saying sudo required in travis.

Reply:
It does. Tried it in my repo + verification passed. Maybe it works by chance? Should I state something extra?